doc/plutus-core-spec/flat-serialisation.tex

```diff
@@ -195,16 +195,17 @@ \subsection{Arrays}
 
 \subsection{Natural numbers}
 We encode natural numbers by splitting their binary representations into
-sequences of 7-bit blocks, then emitting these as a list with the \textbf{least
-  significant block first}:
+sequences of 7-bit blocks, then emitting all but last chunks as a list with
+the \textbf{least significant block first}. The last chunk(most significant chunk)
+will be encoded as regular 7-bit block and appended right after the list of chunks.
 
```

Comment:
I think this needs a sentence like "Every natural number can be written uniquely in the form `$\sum_{k=0}^nk_i2^{7i} with n \geq 0 and k_n \neq 0$`. Using this fact, we define <definition of the encoder> ... ", the point being that we want to (a) disallow an empty sum, and (b) also disallow leading zeros.   The decoder does accept natural numbers encoded with extera leasing zeros (ie, at the end of the list of blocks), but we probably want to specify a canonical way of encoding everything.